### PR TITLE
Assignment07

### PR DESCRIPTION
**Pull request by sonalaggarwal555**
Lesson 07 Assignment
Page 1 Helloworld.vue
All content is from variables in the <script> section of the .vue file.
Page 2 Images.vue
Images are all taken from an array and implemented using a v-for.
Page 3 Accordion.vue
All content comes from an object and is implement using a v-for.
Page 4 grid.vue
A v-if and v-for is used to display content

### CODE REVIEW

File: lesson07/assignment/vue-cli/src/main.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// @import 'node_modules/bootstrap/scss/bootstrap';
+// @import 'node_modules/bootstrap-vue/src/index.scss';
+// @import 'settings-variables';
```
Comment (cherimarie) [inline lesson07/assignment/vue-cli/src/main.scss]: Why are these commented out?

File: lesson07/assignment/vue-cli/src/main.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import App from './App'
+import Router from '/router'
+
+import 'bootstrap';
+import BootstrapVue from 'bootstrap-vue'
+import './main.scss'
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// import { mdbNavbar, mdbNavbarBrand, mdbNavbarToggler, mdbNavbarNav, mdbNavItem, mdbDropdown, mdbDropdownMenu, mdbDropdownToggle, mdbInput, mdbDropdownItem } from 'mdbvue';
+// import { mdbContainer,mdbAccordion } from 'mdbvue';
```
Comment (cherimarie) [inline lesson07/assignment/vue-cli/src/main.js]: No need to keep commented out code. Just delete.

**Comment by sonalaggarwal555** [comment]
Hi Cheri,

I am updating PR with only lesson 7 changes ,will send you shortly .

Thanks,
Sonal

File: lesson07/assignment/vue-cli/src/components/Helloworld.vue [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      features: [
+      {title: 'Feature topic 1', intro:'Feature topic content goes here', image: image1,alt:''},
+      {title: 'Feature topic 2', intro:'Feature topic content goes here', image: image2,alt:''},
+      ]
+    }
+  }
+}
+</script>
+
+<style scooped>
```
Comment (cherimarie) [inline lesson07/assignment/vue-cli/src/components/Helloworld.vue]: Typo: "scooped" -> "scoped" 😄

**Comment by sonalaggarwal555** [comment]
hi Cheri,
Here are changes with less files in pull request.Also i have updated
changes that you mentioned.
https://github.com/sonalaggarwal555/html300/pull/7

Thanks,
Sonal

**Comment by cherimarie** [comment]
@sonalaggarwal555 please make pull requests to your fork, not the main repo. Closing this PR as it cannot be merged. Reopen against your own fork.

File: lesson07/assignment/vue-cli/src/components/Accordian.vue [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <main class="main">
+  <!--Accordion -->
+  <h1>heading</h1>
+  <img src="/assets/cover.jpg" alt="image" class="float-right">
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/Accordian.vue:6]: Remember to keep content in the script portion.

File: lesson07/assignment/vue-cli/src/components/Accordian.vue [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  </div>
+  <!-- Accordion -->
+  </main>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+    items:[{
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/Accordian.vue:37]: 👍

File: lesson07/assignment/vue-cli/src/components/Helloworld.vue [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+</template>
+
+<script>
+const image1 = require('../assets/cover.jpg');
+const image2 = require('../assets/cover2.jpg');
+export default {
+  name: 'app',
+  data () {
+    return {
+      features: [
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/Helloworld.vue:24]: 👍

File: lesson07/assignment/vue-cli/src/components/images.vue [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+    <img src="../assets/pic-1.jpg" alt="..." class="float-right">
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+    aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <!-- Popover-->
+    <p>
+      <b-button class="btn-success" v-b-popover.hover.top="'This is a best place for vacation'" title="Adventure">Did you know?</b-button>
+    </p>
+    <!-- image cards -->
+    <div class="card-columns">
+      <div v-for="card in cards" class="card bg-light text-white">
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/images.vue:14]: 👍

File: lesson07/assignment/vue-cli/src/components/grid.vue [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+  </main>
+</template>
+
+<script>
+const image1 = require('../assets/pic-5.jpg');
+const image2 = require('../assets/pic-6.jpg');
+export default {
+  el: '#app',
+  data () {
+    return {
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/grid.vue:36]: Remember your tab/spacing to make nesting easier to read.

File: lesson07/assignment/vue-cli/src/components/grid.vue [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+        <div v-for:"item in items">
+          <p>{{item.paragraph}}</p>
+        </div>
+      </div>
+      <div class="col-sm-4">
+        <div v-for="sidebar in sidebars">
+          <h2>{{sidebar.title}}</h2>
+          <p><small>{{sidebar.paragraph}}</small></p>
+        </div>
+        <div v-for="sideimage in sideimages">
```
Comment (MadEste) [inline lesson07/assignment/vue-cli/src/components/grid.vue:21]: 👍

**Pull request closed without merging**